sktime/forecasting/varmax.py

```diff
@@ -0,0 +1,299 @@
+# -*- coding: utf-8 -*-
+__all__ = ["VARMAX"]
+__author__ = ["KatieBuc"]
+
+import numpy as np
+import pandas as pd
+from statsmodels.tsa.statespace.varmax import VARMAX as _VARMAX
+
+from sktime.forecasting.base.adapters import _StatsModelsAdapter
+
+
+class VARMAX(_StatsModelsAdapter):
+    """
+    Wrapper for statsmodels VARMAX model
+
+    Vector Autoregressive Moving Average with eXogenous regressors model (VARMAX)
+
+    Parameters
+    ----------
+    y : array_like
+        The observed time-series process :math:`y`, , shaped n_obs x k_endog.
+    X : array_like, optional
+        Array of exogenous regressors, shaped n_obs x k.
```

Comment:
I dont think y and X need to be here. The stuff that goes here are things that user can provide during initialisation. Maybe this should be moved to _fit_forecaster?

```suggestion
```